package.json

```diff
@@ -3,13 +3,16 @@
   "version": "0.38.2",
   "description": "Free-range app utility for IFRAME-based FRAs",
   "type": "module",
-  "exports": "./index.js",
+  "exports": {
+    "./client/client.js": "./client/client.js",
+    "./client/slim-client.js": "./client/slim-client.js",
```

Comment:
Consider `./client/slim.js` to align with the old usage of `ifrau/client/slim`

Reply:
In this case I'll just change the file name entirely to match this. Originally I renamed it because I was developing it side-by-side with the existing code and I just didn't bother changing it back afterwards.